app/javascript/follow-button.js

```diff
@@ -0,0 +1,45 @@
+import jquery from 'jquery'
+window.jQuery = jquery
+window.$ = jquery
+
+$(document).ready(function() {
+  $('.btn-follow').each(function() {
+    var self = $(this);
+    var following = self.data('follows');
+    var styleClass = following ? 'btn-unfollow' : 'btn-follow';
+    self.addClass(styleClass);
+  });
+
+  $(document).on('click', '.btn-follow, .btn-unfollow', function() {
+    var self=$(this);
+    var user_id = self.data('user-id');
```

Comment:
We use camelCase for our `js` files!
```suggestion
    var userId = self.data('user-id');
```